website/StandardAttributes.rst

```diff
@@ -0,0 +1,1388 @@
+         Nepal is at longitude 85.317, latitude 27.717, altitude 1305.
+       </p>
+     </td>
+   </tr>
+
+   </table>
+   </embed>
+     
+
+Camera ID
+=========
+
+These attributes identify the camera. All are optional.
+
+.. raw:: html
+
+   <embed>
+   <table>
+   
+   <tr>
+     <td style="vertical-align: top; width:150px; font-weight:bold"> attribute name</td>
+     <td style="vertical-align: top; width:100px; font-weight:bold"> type </td>
+     <td style="vertical-align: top; width:500px; font-weight:bold"> definition </td>
+   </tr>
+
+   <tr style="border-width:3px">
+     <td style="vertical-align: top; width:150px"> <tt> <b> cameraMake </b> </tt>
+     </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Manufacturer or vendor of the camera. If present, the value should
+         be UTF-8-encoded and have a nonzero length.
+      </p>
+     </td>
+   </tr>
+   
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> cameraModel </b>
+     </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Model name or model number of the camera.  If present, the value
+         should be UTF-8-encoded and have a nonzero length.
+      </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> cameraSerialNumber
+     </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Serial number of the camera If present, the value should be
+         UTF-8-encoded and have a nonzero length.  Note that despite the
+         name, the value can include non-digits as well as digits.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> cameraFirmwareVersion
+     </b> </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         The firmware version of the camera. If present, the value should be
+         UTF-8-encoded and have a nonzero length.
+       </p>
+     </td>
+   </tr>
+
+   <tr>
+     <td style="vertical-align: top; width:150px"> <tt> <b> cameraUuid </b>
+     </tt> </td>
+     <td style="vertical-align: top; width:100px"> <tt> string </tt> </td>
+     <td style="vertical-align: top; width:500px">
+       <p style="padding-bottom:15px">
+         Identifies this camera uniquely among all cameras from all vendors.
+       </p>
+       <p style="padding-bottom:15px">
+         Uniqueness could be accomplished with, e.g., a MAC address, a
+         concatenation of cameraMake, cameraModel, cameraSerialNumber, etc.
```

Comment:
The string can have arbitrary format, right? It doesn't need to follow the UUID 128 bit string format, even though that's implied by the name? It may be worth clarifying

Reply:
done.